.github/workflows/msbuild.yml

```diff
@@ -45,7 +45,10 @@ jobs:
         ref: v${{ steps.regex-match.outputs.group1 }}
     - name: Build Dafny with local Boogie
       working-directory: dafny
-      run: dotnet build Source/Dafny.sln
+      run: |
+        dotnet build Source/Dafny.sln
+        dotnet build -f net462 Source/DafnyRuntime/DafnyRuntime.csproj
+        dotnet build -f netstandard2.0 Source/DafnyRuntime/DafnyRuntime.csproj
```

Comment:
Does this generate 3 runtimes?

Reply:
Yeah, one for each of those frameworks, so that folks downloading from NuGet can get the one that works for them. My understanding is that this is the best way to make .NET libraries available for many versions, but I'd love to hear about a better way if there is one.